modules/core/include/opencv2/core/saturate.hpp

```diff
@@ -58,7 +58,7 @@ namespace cv
 
 /** @brief Template function for accurate conversion from one primitive type to another.
 
- The functions saturate_cast resemble the standard C++ cast operations, such as static_cast\<T\>()
+ The function saturate_cast resemble the standard C++ cast operations, such as static_cast\<T\>()
```

Comment:
I can't say for sure whether it is better to use singular or plural here. But anyway it should be uniform with the latter text, so either there should be plural or `they` should be replaced with `it`in the latter text.